amp/repository/aim/view/viewActivityDifferences.jsp

```diff
@@ -275,14 +279,17 @@ setHoveredTable("dataTable", true);
 setHoveredRow("rowHighlight");
 
 if(document.getElementById('method').value == "enableMerge") {
-	document.getElementById('mergeButton').disabled = "disabled";
-	document.getElementById('mergeButton').style.display = 'none';
-	document.getElementById('saveButton').disabled = "";
-	document.getElementById('saveButton').style.display = 'block';
-} else {
-	document.getElementById('saveButton').disabled = "disabled";
-	document.getElementById('saveButton').style.display = 'none';
+    document.getElementById('mergeButton').disabled = "disabled";
+    document.getElementById('mergeButton').style.display = 'none';
+    document.getElementById('saveButton').disabled = "";
+    document.getElementById('saveButton').style.display = 'block';
+} else if (document.aimCompareActivityVersionsForm.method.value === "viewDifferences"){

```

Comment:
this is fine given the current approach, but I will explain why it would be better to show the buttons or hide them server side in our next call